packages/core/src/browser/keybinding.ts

```diff
@@ -60,7 +60,7 @@ export interface ScopedKeybinding extends common.Keybinding {
 
 export const KeybindingContribution = Symbol('KeybindingContribution');
 /**
- * Representation of a keybinding contribution.
+ * Allows extensions to contribute {@link common.Keybinding}s
```

Comment:
@JonasHelming out of curiosity, which IDE or text editor are you using when preparing the changes?
I know that the `{@link ...`} syntax is not supported at the moment by TypeScript so it does not render correctly when using Theia or VS Code to develop: https://github.com/microsoft/TypeScript/issues/35524

Reply:
It does not render properly in VS Code/Theia, but linking works as expected in the generated doc (using `yarn docs`).
I agree that it is a bit noisy in the IDE and `@link` should definitely be avoided when describing `params`. I guess most people rather look at the documentation directly in the IDE so maybe it's better to omit the `@link` tags.

Reply:
I see, it helps when generating the documentation: https://eclipse-theia.github.io/theia/docs/next/interfaces/filesystem.fileservicecontribution.html (previous documentation pull-request) :)

![image](https://user-images.githubusercontent.com/40359487/96449526-f8d63100-11e2-11eb-9318-b3aae0e88d2f.png)

I think these links are definitely worthwhile :) 
My only suggestion would be to add `@link` for the first occurrence of the word rather than each occurrence in the same documentation block. I think this would be the best tradeoff, the links would still be present and in the code it would not be as noisy.

Reply:
Applied in the current PR